query/promql/promql.go

```diff
@@ -23,42 +21,44 @@ import (
 
 // DO NOT EDIT: This file is auto generated by the pigeon PEG parser generator.
 
+//lint:file-ignore SA6001 Ignore all unused code, it's generated
```

Comment:
Wow it did not occur to me that you could just do this but it's so obvious it hurts me to see that I didn't realize it.